webcompat/views.py

```diff
@@ -41,14 +42,14 @@
 from webcompat.helpers import is_valid_issue_form
 from webcompat.helpers import prepare_form
 from webcompat.helpers import set_referer
-from webcompat.issues import report_issue
 from webcompat.helpers import get_extra_labels
 from webcompat.helpers import mockable_response
+from webcompat.issues import report_issue
+from webcompat.templates import bust_cache
```

Comment:
nit: this one is below. ;)

Reply:
(@karlcow -- sorry, I don't follow. do you mean `from webcompat.templates import bust_cache` should be moved down below?)

Reply:
yes sorry that was unclear, though at the same time, I see that the order is a bit messed up for other things. 

Let's do that later.